package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "api.webmaker.org",
+  "version": "0.0.0",
+  "description": "Services for Webmaker",
+  "main": "server.js",
+  "scripts": {
+    "start": "node server.js",
+    "test": "npm run test:api && npm run lint",
+    "test:api": "lab -c --verbose --colors --assert code --timeout 5000",
```

Comment:
You should also add `-t 100` which sets the minimum code coverage percentage to 100%